pkg/service/v1alpha1/node.go

```diff
@@ -283,16 +283,11 @@ func (ns *node) NodeUnpublishVolume(
 	defer removeVolumeFromTransitionList(volumeID)
 
 	notMnt, err := ns.mounter.IsLikelyNotMountPoint(target)
-	if err != nil && !os.IsNotExist(err) {
+	if (err == nil && notMnt) || os.IsNotExist(err) {
```

Comment:
what if err is nil and notMnt is false?  does os.IsNotExist(err) checks that err can be nil also?

Reply:
this is taken care of inside IsNotExist function